### DISCUSSION TITLE
How to build conductor with specific version?

### DISCUSSION THREAD

**Discussion by ykes60513** [Netflix/conductor#2970] [Q&A]
It seems a silly question, but I really have no idea how to build it with like 3.5.3 release locally.
And I am using M1 chip Mac...in case if there is any different step, may need to change
the Elastic Search version manually??

**Comment by apanicker-nflx** [comment]
Please see the response [here](https://github.com/Netflix/conductor/issues/2975#issuecomment-1121610967)
